openslides/core/apps.py

```diff
@@ -110,17 +110,13 @@ def get_angular_constants(self):
                     items=[]))
             # Add the config variable to the current group and subgroup.
             config_groups[-1]['subgroups'][-1]['items'].append(config_variable.data)
-        config_variables = {
-            'name': 'OpenSlidesConfigVariables',
-            'value': config_groups}
+        constants['OpenSlidesConfigVariables'] = config_groups
 
         # Send the privacy policy to the client. A user should view them, even he is
         # not logged in (so does not have the config values yet).
-        privacy_policy = {
-            'name': 'PrivacyPolicy',
-            'value': config['general_event_privacy_policy']}
+        constants['PrivacyPolicy'] = config['general_event_privacy_policy']
```

Comment:
Remove this config var and create a privacy policy at another place. (See the comment in consumers.py).